now/executor/gateway/playground/playground.py

```diff
@@ -439,7 +400,7 @@ def customize_semantic_scores():
             value=0.5,
             key='weight_' + str(i),
         )
-        st.session_state.semantic_scores[f'{i}'] = (q_field, id_field, enc, w)
+        st.session_state.score_calculation[f'{i}'] = (q_field, id_field, enc, w)
```

Comment:
```suggestion
        st.session_state.score_calculation[f'{i}'] = [q_field, id_field, enc, w]
```
Without this change, and with conversion to list removed in bff, I imaging this should be breaking custom score calculation in the playground?